bears/natural_language/WriteGoodLintBear.py

```diff
@@ -0,0 +1,78 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='write-good',
+        output_format='regex',
+        output_regex=r'(?P<message>.*)\s*on\s*line\s*(?P<line>\d+)\s*at\scolumn'
+                      '\s*(?P<column>\d+)'
+        )
+class WriteGoodLintBear:
+    """
+    Lints the text files using ``write-good`` for improving proses.
+    """
+    LANGUAGES = {"Natural Language"}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         passive: bool=False,
+                         illusion: bool=False,
+                         adverb: bool=False,
+                         so: bool=False,
+                         thereIs: bool=False,
+                         weasel: bool=False,
+                         tooWordy: bool=False,
+                         cliches: bool=False):
+        """
+        Bear configuration arguments.
+        Using 'False' will disable the check.
+
+        :param passive:        Checks for passive voice.
+
+        :param so:             Checks for `so` at the beginning of the sentence.
```

Comment:
i guess it would be ``So``, as it would be capitalized if at beginning of the sentene :D